Source/CLI/rawcooked.1

```diff
@@ -2,9 +2,12 @@
 .SH NAME
 \fBRAWcooked\fR \- encode and decode audio\-visual RAW data with Matroska, FFV1 and FLAC
 .SH SYNOPSIS
-.
-.SS
 \fBrawcooked \fIfolder\fR | \fIfile
+.SH DESCRIPTION
+.TP
+\fBRAWcooked\fR encodes audio\-visual RAW data into the Matroska container (MKV), using the video codec FFV1 for the image and audio codec FLAC for the sound\. The metadata accompanying the RAW data are preserved, and sidecar files, like MD5, LUT or XML, can be added into the Matroska container as attachments\. This allows for the management of these audio\-visual file formats in an effective and transparent way (e\.g\. native playback in VLC), while saving typically between one to two thirds of the needed storage, and speeding up file writing and reading (e\.g\. to/from harddisk, over network or for backup on LTO)\.
```

Comment:
Perhaps "the audio codec" for consistency?

Reply:
I’m always happy for better English than mine… Thank you, @kieranjol!

Reply:
FYI: More options will follow in a couple of minutes.